pkg/config/images.go

```diff
@@ -80,36 +78,34 @@ func selectContainerImagesOSA(cs *acsapi.OpenShiftManagedCluster) {
 	switch cs.Properties.OpenShiftVersion {
 	//TODO: confirm minor version after release
 	case "v3.10":
-		v := "v3.10.14" // TODO: perhaps we should calculate this from c.ImageVersion
-		c.ControlPlaneImage = image(cs, "control-plane", v)
-		c.NodeImage = image(cs, "node", v)
-		c.ServiceCatalogImage = image(cs, "service-catalog", v)
-		c.AnsibleServiceBrokerImage = image(cs, "ansible-service-broker", v)
-		c.TemplateServiceBrokerImage = image(cs, "template-service-broker", v)
-		c.RegistryImage = image(cs, "docker-registry", v)
-		c.RouterImage = image(cs, "haproxy-router", v)
-		c.WebConsoleImage = image(cs, "web-console", v)
-
-		c.MasterEtcdImage = "registry.access.redhat.com/rhel7/etcd:3.2.22"
-		c.EtcdOperatorImage = "quay.io/coreos/etcd-operator:v0.9.2"
-		c.KubeStateMetricsImage = "quay.io/coreos/kube-state-metrics:v1.4.0"
-		c.AddonsResizerImage = "k8s.gcr.io/addon-resizer:1.7"
-
-		c.OAuthProxyImage = "registry.access.redhat.com/openshift3/oauth-proxy:" + v
-		c.PrometheusImage = "registry.access.redhat.com/openshift3/prometheus:" + v
-		c.PrometheusAlertBufferImage = "registry.access.redhat.com/openshift3/prometheus-alert-buffer:" + v
-		c.PrometheusAlertManagerImage = "registry.access.redhat.com/openshift3/prometheus-alertmanager:" + v
-		c.PrometheusNodeExporterImage = "registry.access.redhat.com/openshift3/prometheus-node-exporter:" + v
-
-		c.RegistryConsoleImage = "registry.access.redhat.com/openshift3/registry-console:" + v
-
-		c.AzureCLIImage = "docker.io/microsoft/azure-cli:latest" //TODO: create mapping for OSA release to any other image we use
-
-		if c.SyncImage == "" {
-			c.SyncImage = "quay.io/openshift-on-azure/sync:v3.10"
+		v := "v3.10.14" // TODO: perhaps we should calculate this from c.Images.ImageVersion
```

Comment:
The latest image that we have produced is v3.10.34.  Is this something we should fix?

Reply:
yes :( https://github.com/openshift/openshift-azure/issues/465 - please can you pick that up @kwoodson ?